cms/variant-migrations/migrations/20240708184150-update-models-gdc-v2-sequence-urls.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  async up(db) {
+    const collection = db.collection('models');
+
+    const cursor = collection.find({
+      source_sequence_url: {
+        $regex: /https:\/\/portal\.gdc\.cancer\.gov\/v1\/repository\?facetTab=files&filters=%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22cases\.case_id%22%2C%22value%22%3A%5B%22(.*?)%22%5D%7D%7D%5D%7D&searchTableTab=files/,
+      },
+    });
+
+    while (await cursor.hasNext()) {
+      const doc = await cursor.next();
+      const pattern = /https:\/\/portal\.gdc\.cancer\.gov\/v1\/repository\?facetTab=files&filters=%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22cases\.case_id%22%2C%22value%22%3A%5B%22(.*?)%22%5D%7D%7D%5D%7D&searchTableTab=files/;
```

Comment:
Not a problem per se, but you have this sequence repeated, how confident are you that it is exactly the same as the one above without any typos? Probably should've reused a variable. Since migrations likely won't need future editing we can keep it like this if its working.